src/app/loginComponents/requests/requests.component.ts

```diff
@@ -75,6 +77,18 @@ export class RequestsComponent implements OnInit {
     this.isAdmin$ = this.userQuery.isAdmin$;
     this.isCurator$ = this.userQuery.isCurator$;
     this.userId$ = this.userQuery.userId$;
+
+    this.isAdmin$.pipe(takeUntil(this.ngUnsubscribe)).subscribe(isAdmin => {
```

Comment:
generally, subscribing within another subscribe is a bad sign.  In this case, you can use something like 
```
combineLatest(this.admin$, this.isCurator$).pipe(takeUntil(this.ngUnsubscribe)).subscribe(([isAdmin, isCurator]:[boolean, boolean]) => {
// do stuff
});} 
```

Reply:
better yet, put the combined observable inside the userQuery as isAdminOrCurator$ and then subscribe to that and do stuff.